docs/streaming/hls.md

```diff
@@ -45,3 +45,22 @@ For example:
     <![CDATA[ https://example.com/stream_hls.m3u8 ]]>
 </MediaFile>
 ```
+
+# Customizing HLS
+
+Fluid Player has hooks that support `hls.js` configuration, a full list of configurable properties can be found in the [official `hsl.js` API docs](https://github.com/video-dev/hls.js/blob/master/docs/API.md).
+Below is an example of a configuration that changes the maximum buffer length and the starting quality of a stream.
```

Comment:
Below is an example of a configuration that changes the maximum buffer length and streaming quality of the video.